packages/subiquity_client/lib/src/http_unix_client.dart

```diff
@@ -60,7 +66,15 @@ class HttpUnixClient extends BaseClient {
     if (_socket == null) {
       var address = InternetAddress(path, type: InternetAddressType.unix);
       _socket = await Socket.connect(address, 0);
-      _socket?.listen(_processData);
+
+      final keepalive =
+          RawSocketOption.fromBool(SOL_SOCKET, SO_KEEPALIVE, true);
+      _socket!.setRawOption(keepalive);
+
+      final keepintvl = RawSocketOption.fromInt(IPPROTO_TCP, TCP_KEEPINTVL, 60);
```

Comment:
Hard to see what happens under the hood in Dart, but I get "Operation not supported" with a minimal C test app:
```
int keepintvl = 10;
int rc = setsockopt(fd, IPPROTO_TCP, TCP_KEEPINTVL, &keepintvl, sizeof(int));
printf("TCP_KEEPINTVL: %d (%s)\n", rc, strerror(errno)); // TCP_KEEPINTVL: -1 (Operation not supported)
```